src/main/java/ch/heigvd/dil/data_structures/Site.java

```
@@ -0,0 +1,64 @@
+package ch.heigvd.dil.data_structures;
+
+import java.util.List;
+
+/** Représente un site. */
+public class Site {
+  private Site.Config config;
+  private final String path;
+
+  /**
+   * @param config La configuration du site
+   * @param path Le chemin source du site
+   */
+  public Site(Site.Config config, String path) {
+    this.config = config;
+    this.path = path;
+  }
+
+  /**
+   * @return Les pages contenues dans le site
+   */
+  public List<Page> retrievePages() {
+    return null;
```

You can throw unsupportef method exception rather than return null